src/main/scala/uk/gov/hmrc/hmrcfrontend/views/implicits/RichAccountMenuSupport.scala

```diff
@@ -44,6 +44,15 @@ trait RichAccountMenuSupport extends {
         yourProfile = accountMenu.yourProfile.copy(
           href = updateDefault(accountMenu.yourProfile.href, accountMenuConfig.yourProfileHref)
         ),
+        businessTaxAccount = accountMenu.businessTaxAccount match {
```

Comment:
we could make this a map too, like https://github.com/hmrc/play-frontend-hmrc/pull/151/files#r762849997 but totally optional